fastapi_pagination/ext/pymongo.py

```diff
@@ -0,0 +1,25 @@
+from typing import Any, Dict, List, Optional
+
+
+from pymongo.collection import Collection
+
+
+from ..api import create_page, resolve_params
+from ..bases import AbstractPage, AbstractParams
+
+
+def paginate(
+    collection: Collection,
+    query_filter: Optional[Dict[Any, Any]] = None,
+    params: Optional[AbstractParams] = None,
+    **kwargs: Any
+) -> AbstractPage:
+    params = resolve_params(params)
+    query_filter = query_filter or {}
+
+    raw_params = params.to_raw_params()
+    total = collection.count_documents(query_filter)
+    cursor = collection.find(query_filter, skip=raw_params.offset, limit=raw_params.limit, **kwargs)
```

Comment:
I think you lose the ability to call `.where` if you manually call `.find` here, right?

Reply:
Hey @letoss,

thanks for your feedback. I have the feeling I didn't get the question. As far as i know there is no `.where` function on the collection type, but I think building the query like that would solve your problem?
```python
db.student.find( { $where: function() { return (this.english == this.science) }})
```
Could you provide me a link to the docs of the `.where` function so I can have a look and really understand your point?
I used the motor integration as a "template" for my integration and it uses the `.find` function too.
Maybe we also need to change this integration.
Thanks in advance!

Reply:
Hello, @nikstuckenbrock .

It took me a moment to find the method again. It's from a `cursor`, not collection. Because of it, I was saying that it could be a good idea for the pagination to receive a Collection or the cursor directly.

This is the link: https://pymongo.readthedocs.io/en/stable/api/pymongo/cursor.html?highlight=.where#pymongo.cursor.Cursor.where

Your code snippet totally works in the case I'm thinking. You can ignore completely my comment.

Reply:
Hello @letoss,

it's a valid point you mentioned there.
Maybe a new feature for the future.
Thansk for your feedback and the conversation!